demo/campaign/src/components/Root.tsx

```diff
@@ -33,6 +36,8 @@ export const Root: React.FC<Props> = ({ children, title }) => {
         <Mjml>
             <MjmlHead>
                 {!!title && <MjmlTitle>{title}</MjmlTitle>}
```

Comment:
The text inside `Typography` and also the `Divider` have a side-padding by default, in addition to the padding created by `IndentedSectionGroup`. 

<img width="772" alt="image" src="https://github.com/user-attachments/assets/ea9cdd0d-e2b6-4a4d-8be3-6dfde6565026">

It should probably only include the padding from `IndentedSectionGroup`, right?
You could remove the default-padding from the Mjml components by adding the following to `MjmlHead`. 

```tsx
<MjmlAttributes>
    <MjmlText padding={0} />
    <MjmlDivider padding={0} />
</MjmlAttributes>
```